views/layout.pug

```diff
@@ -11,6 +11,7 @@ html(lang="en")
     meta(name='keywords', content='ethereum, explorer, blockchain, etherchain, cryptocurrency, cryptocurrencies, bitcoin')
 
     title ewasm testnet
+    link(rel='shortcut icon' href='/explorer/images/favicon.ico')
```

Comment:
Thanks! I think lately favicon's don't need to be this horrible tiny ICO format, but can be larger PNGs.